daffodil-cli/NOTICE-bin

```diff
@@ -0,0 +1,81 @@
+Apache Daffodil
+Copyright 2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+Based on source code originally developed by
+- The Univerisity of Illinois National Center for Supercomputing Applications (http://www.ncsa.illinois.edu/)
+- Tresys Technology (http://www.tresys.com/)
+- International Business Machines Corporation (http://www.ibm.com)
+
+******************
```

Comment:
Do not list out licenses in the NOTICE file.

Reply:
I took this layout from another Apache project (can't remember which one). Looking around it seems every project has their own variant. It would be really nice if there were an ASF provided template that shows the expected format of these files. It seems the LICENSE/NOTICE files are some of the biggest hangups for incubators, and I think part of that is the lack of resources.

Simple enough change to make though. Will have a fixup shortly.